### PR TITLE
Login form into storyCMS

### PR DESCRIPTION
**Pull request by karimkefi**
Adding Login form basic to storyCMS

### CODE REVIEW

File: checkAdmin.php [hunk 1]
```diff
@@ -1,34 +1,59 @@
...
+/**
+ *The function takes 2 values (Admin user and password) and checks if the pair match fixed values
+ *
+ *@$user string value
+ *@$passWord string value
+ *
+ *@return boolean result to confirm correct details
+ */
 function userCredentials ($user, $passWord) {
     if ($user == 'kefi' && $passWord == 'A1') {
```
Comment (kyam-mayden) [inline checkAdmin.php]: should use '===' as this checks for type too

**Review by kyam-mayden — CHANGES_REQUESTED** [review]
Only one thing to look at

**Review by kyam-mayden — APPROVED** [review]
Love it

File: adminCheck.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+if ($boolResult) {
+    $_SESSION['userLoggedIn'] = true;
+    header("Location: cmsHomePage.php");
+} else {
+    $_SESSION['userLoggedIn'] = false;
+    $_SESSION['invalidcombo'] = true;
+    header("Location: adminLogin.php");
+}
+
+
```
Comment (JamesCullen2) [inline adminCheck.php:21]: white space

File: adminFunctions.php [hunk 3]
```diff
@@ -39,24 +27,33 @@ function isValidString ($stringInput) {
...
+    $db = new PDO('mysql:host=127.0.0.1; dbname=karimPortfolioCMS', 'root');
 
-$userName = $_POST['InputUserName'];
-$userPassword = $_POST['InputPswd'];
+    $query = $db->prepare("SELECT `herbs` FROM `mango` WHERE `fruits` = :uName;");
+
+    $query->bindParam(':uName', $enteredUName);
+    $query->execute();
+    $passwordDB = $query->fetch();
+
```
Comment (JamesCullen2) [inline adminFunctions.php:53]: white space

File: test/adminFunctions.php [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+    //Malformed
+    public function test_sanitiseString_Malformed()
+    {
+        $inputA = ['aa', 12];
+        $this->expectException(TypeError::class);
+        sanitiseString($inputA);
+    }
+
```
Comment (JamesCullen2) [inline test/adminFunctions.php:68]: white space

**Review by JamesCullen2 — COMMENTED** [review]
All good !

**Pull request merged**